src/client.ts

```diff
@@ -0,0 +1,132 @@
+import * as request from "request-promise-native";
+
+interface assoc {
+    [key: string]: any;
+}
+
+export class Client {
+
+    endpoint: string = 'https://appwrite.io/v1';
+    headers: assoc = {
+        'content-type': '',
+        'x-sdk-version': 'appwrite:typescript:',
+    };
+    
+    /**
+     * Set Project
+     *
+     * Your project ID
+     *
+     * @param string value
+     *
+     * @return self
```

Comment:
these don't give any useful information to the user of the API that can't be inferred from the types - as an outsider using this library, I'd see a `.d.ts` file with something like:
```ts
class Client {
  ...
  
  /**
   * <your comment>
   */
  setProject(value: string): this;
}
```

So the explanations for what are the types of `value` (`@param string value`) and `@return self` don't add any information not already present in the types themselves.